### ISSUE TITLE
This entire things unfortunately doesn't work!

### ISSUE THREAD

**Issue by knaggit** [getkirby-solutions/heroku#1]
Hello!
I am struggle with deploying Kirby on Heroku actually. After a while I discovered this repo and even tried after a while to just push this repo only to Heroku. It doesn't work. There are no logs I can give because nothing throwed anything.

Problem:
The panel isn't working. Every time I try to login I get redirected to the login page again without a result. What can I do? Where can I help? Is it possible to solve that problem?

**Comment by hrstkatomas** [comment]
We managed to get Kirby running on Heroku, but there is still huge problem:
**Every time you push to projects git repository or the dyno restarts, your entire content disappears.**

Like many modern hosting solutions based on containers (such as Docker), the Heroku filesystem is 
[ephemeral](https://devcenter.heroku.com/articles/dynos#ephemeral-filesystem) - that means that any changes to the filesystem whilst the dyno is running only last until that dyno is shut down or restarted.

**tl;dr:** Files shouldn't be used for permanent storage

To use file based CMS Heroku advises to use Amazon S3 to store your content or for WordPress there is [this plugin](https://github.com/dzuelke/wordpress-12factor#twelve-factor-wordpress) that can store your content in folder with installed packages (similar idea could be used with Kirby). Either way seems like too much of a hassle.
